script.js

```diff
@@ -0,0 +1,38 @@
+let country = 'Moldova';
+let continent = 'Europe';
+let population = 1.2;
+let isIsland = false;
+let countryDescription = `${country} is located in ${continent}, ${population} million people live here`;
+
+console.log(country);
+console.log(continent);
+console.log(population);
+console.log(countryDescription);
+
+
+
+
+let weightSarah = 65;
+let weightBob = 93;
+let heightSarah = 1.75;
+let heightBob = 1.89;
+
+let sarahBMI = weightSarah / (heightSarah ** 2);
+console.log(sarahBMI);
+
+let bobBMI = weightBob / (heightBob ** 2);
+console.log(bobBMI);
+
+if (sarahBMI < 18.5) {
+    console.log("Sarah has underweight");
+}
+
+if (sarahBMI > 18.5 && sarahBMI < 25) {
```

Comment:
You have only 2 conditions here and for BOB you made it right so it is `if (condition1) {} else {condition2}` your overcomplicated it

Reply:
Technically it is correct. But I agree, you could use `if else` form